### PR TITLE
Azure Event Hubs October Releases

### PR DESCRIPTION
**Pull request by antkmsft**

### CODE REVIEW

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0-beta.3 (2023-10-10)
 
 ## 1.0.0-beta.2 (2023-09-12)
```
Comment (antkmsft) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:5]: @LarryOsterman, we need a changelog entry here.

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-## 1.0.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0-beta.3 (2023-10-10)
```
Comment (LarryOsterman) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.3 (2023-10-10)

Split the processor blob checkpoint store into its own package (azure-messaging-eventhubs-checkpointstore-blob).

```

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md (hunk 1 above)
Comment (LarryOsterman) [reply sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md]: Added.

File: sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,8 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.1 (2023-10-10)
 
 ### Features Added
 
 - Initial release. Split from the `azure-messaging-eventubs` package.
```
Comment (LarryOsterman) [inline sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md:7]:
```suggestion
- Initial release. Split from the `azure-messaging-eventubs-cpp` package.
```

**Review by LarryOsterman — APPROVED** [review]

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2023-10-10)
 
-### Features Added
-
-### Breaking Changes
+Split the processor blob checkpoint store into its own package (azure-messaging-eventhubs-checkpointstore-blob).
```
Comment (LarryOsterman) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:5]:
```suggestion
* Removed all direct dependencies on Azure Blob Storage and moved those dependencies into a helper package 
(azure-messaging-eventhubs-checkpointstore-blob-cpp).
```

**Review by LarryOsterman — APPROVED** [review]

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md (hunk 1 above)
Comment (antkmsft) [reply sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md]: Thank you!

**Pull request merged**